prep-css-box-model/style.css

```diff
@@ -0,0 +1,13 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+  margin: 20px;
+  padding: 10px;
+  border: 3px, double, darkgray;
```

Comment:
Hi Tim, you border styling isn't being applied due to the commas here, try removing them.